Tokenization_Samples/python_samples/README.md

```diff
@@ -0,0 +1,90 @@
+# Tokenization Samples
+
+## Prerequesties
+
+- Python v2.7.x up.
+- rfc3339 python library.
+- requests python library.
+- eventlet python library.
+- Access to CipherTrust Manager.
+- Tokengroup and Tokentemplate present in CipherTrust Manager.
+
```

Comment:
Python Samples have samples for Tokenize and Detokenize while Java samples have samples for "Tokenize/detokenize" and "Tokenize/detokenize Bulk mode". Can we please make this symmetric for both. 

Review comments for Java sample page( As I cant find link to add review comments for it):

- Title to be "JavaTokenization Samples"
- Font correction for "Output"
- Move REST API link to main page and remove from each